### DISCUSSION TITLE
authentication

### DISCUSSION THREAD

**Discussion by adeelumar-it** [community/community#142575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i have lost mobile phone where i was using git hub authenticator for login.
now what i have to do?

**Comment by Aninooo** [comment]
If you've lost your phone with the GitHub authenticator, here are some steps you can take:

Use recovery codes that you saved when setting up two-factor authentication (2FA). These codes can help you log in.

Check if you set up SMS as a backup option. If you did, you can receive a code via text.

If you can’t access recovery codes or SMS, contact [GitHub Support](https://support.github.com/) for assistance. They can help you regain access to your account.

**Comment by thomassiard** [comment]
1. Use Your Recovery Codes
When you set up 2FA on GitHub, you were provided with a set of recovery codes. Check if you saved them (often recommended to save as a file, print, or securely store them).
Use one of these recovery codes to log in:
On the 2FA screen, select "Enter a recovery code."
Enter one of your saved recovery codes to bypass 2FA.
2. Try Other Backup Methods
If you set up a secondary 2FA method (like SMS-based codes, or a backup authenticator app on another device), try using that to log in.
3. Contact GitHub Support
If you don’t have recovery codes or a backup method, you’ll need to reach out to GitHub Support to regain access to your account:
Go to [GitHub Support](https://support.github.com/).
Choose Contact GitHub Support and describe your situation in detail.
They may ask for verification details to confirm your identity.
